float64 target_x
float64 target_y
bool is_active
bool enable_dodge
---
bool success 